types/react-resizable/index.d.ts

```diff
@@ -0,0 +1,65 @@
+// Type definitions for react-resizable 1.7.1
+// Project: https://github.com/STRML/react-resizable
+// Definitions by: Tang Xinyao <https://github.com/tangxinyao>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.4
+
+import * as React from "react";
+
+type SyntheticKeyboardEvent = React.KeyboardEvent<{}>;
+type SyntheticEvent = React.SyntheticEvent<{}>;
+
+declare module "react-resizable" {
```

Comment:
Can you unnest the contents of this ambient module declaration? The only reason not to is that Synthetic[Keyboard]Event are outside, but I don't see that they are used anywhere in any case.